Core/clim-basic/sheets.lisp

```diff
@@ -779,15 +779,19 @@ might be different from the sheet's native region."
               (sheet-transformation sheet)))))
     native-transformation))
 
-;;; Named sheets
+;;; Top-level sheets
 
-(defclass named-sheet-mixin ()
-  (;; The NAME slot intentionally uses the same slot name as the NAME
+(defclass top-level-sheet-mixin ()
+  ((focused-sheet :initform nil :accessor focused-sheet)
```

Comment:
This slot doesn't seem to be used anywhere (within this pull request).

Reply:
fixed